scripts/Dockerfile.production

```diff
@@ -0,0 +1,44 @@
+FROM golang:1.11-stretch AS builder
+
+ARG BUILD=release
+
+WORKDIR /go/src/github.com/cozy/cozy-stack
+COPY . .
+RUN set -eux && \
+    go get -d -v ./... && \
+    bash -ex ./scripts/build.sh ${BUILD} && \
+    rm ./cozy-stack*.sha256
+
+
+
+FROM node:8-stretch-slim
+
+ENV COUCHDB_PROTOCOL=http \
+    COUCHDB_HOST=couchdb \
+    COUCHDB_PORT=5984 \
+    COUCHDB_USER=cozy \
+    COUCHDB_PASSWORD=cozy \
+    COZY_PASSWORD_RESET_INTERVAL=0
```

Comment:
Why do you set `0` to `password_reset_interval`?

Reply:
No real reason. 15 will be better I guess ?

Reply:
In that case, I'd prefer to not set it in the configuration file and let the stack choose its default.

Reply:
Ok, removed